src/Roots.jl

```diff
@@ -285,14 +271,48 @@ function make_rhs(model::RootsModel)
         dY.vegetation.rwc[1] = flow_out_roots - flow_out_stem
         dY.vegetation.rwc[2] =
             flow_out_stem -
-            compute_transpiration(model.configuration,t)
+            compute_transpiration(model.configuration.transpiration, t)
```

Comment:
I think having `compute_...` as a prefix is redundant, what do you think?

Reply:
sure! Ill change them